src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -85,9 +86,15 @@ class Submitter(stagingBaseDir: File,
     // EASY-1464 3.3.7 checksums
     _ <- samePayloadManifestEntries(stageBag, draftBag)
     // EASY-1464 step 3.3.9 Move copy to submit-to area
+    _ = stageBag.baseDir.parent.list.foreach(setRights(_))
     _ = stageBag.baseDir.parent.moveTo(submitDir)
   } yield ()
 
+  private def setRights(file: File) = {
+    file.addPermission(PosixFilePermission.GROUP_WRITE)
+    file.setGroup("deposits")
```

Comment:
Causes `java.nio.file.attribute.UserPrincipalNotFoundException` in unit tests.

Reply:
I vaguely remember having seen similar things in `easy-split-multi-deposit`. Maybe you could take a look at: https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/73189001217c2bf31b487eb8356f76ea4e9ffc31/src/main/scala/nl.knaw.dans.easy.multideposit/actions/SetDepositPermissions.scala#L72-L90